src/lib/utils/thorchain/hooks/useIsThorchainLpDepositEnabled.tsx

```diff
@@ -0,0 +1,29 @@
+import type { AssetId } from '@shapeshiftoss/caip'
+import { assetIdToPoolAssetId } from '@shapeshiftoss/swapper'
+
+import { useThorchainMimir } from './useThorchainMimir'
+
+export const useIsLpDepositEnabled = (assetId: AssetId | undefined) => {
```

Comment:
Note that *only* relates to the `PAUSELPDEPOSIT-` mimirs - a pool may be disabled for other reasons (global LP, chain halted etc) but that's out of the realm of this which checks explicitly for deposit pause mimirs